tests/api/v2/test_user_groups.py

```diff
@@ -0,0 +1,50 @@
+
+# -*- coding: utf-8 -*-
+"""
+Test suite for the community-developed Python SDK for interacting with Lacework APIs.
+"""
+
+import pytest
+
+from laceworksdk.api.v2.user_groups import UserGroupsAPI
+from laceworksdk.exceptions import ApiError
+from tests.api.test_base_endpoint import BaseEndpoint
+from laceworksdk import LaceworkClient
+
+
+# Tests
+
+@pytest.fixture(scope="module")
+def api_object(api):
+    return api.user_groups
+
+@pytest.fixture(scope="module")
+def test_user():
+     c = LaceworkClient()
+     res = c.team_users.create("test", "noreply@lacework.com", "test")
+     guid = res["data"]["userGuid"]
+     yield guid
+     c.team_users.delete(guid)
```

Comment:
I actually really like this, but can we just use the existing `api` object rather than creating another `LaceworkClient`?
```suggestion
def test_user(api):
     res = api.team_users.create("test", "noreply@lacework.com", "test")
     guid = res["data"]["userGuid"]
     yield guid
     api.team_users.delete(guid)
```